students/ZhuojingXie/lesson04/dict_lab.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+
+#Dictionaries 1
+d1 = {
+    'name': 'Chris',
+    'city': 'Seattle',
+    'cake': 'Chocolate'
+}
+print(d1)
+
+d1.pop('cake')
+print(d1)
+
+d1['fruit'] = 'Mango'
+print(d1.keys())
+print(d1.values())
+print('cake' in d1.keys())
+
+
+#Dictionaries 2
+d2 = d1.copy()
+for key in d2:
+    d2[key] = d2[key].lower().count('t')
```

Comment:
You can do `for key, value in d2.items()` to access both keys and values directly in your loop